release-notes/opensearch-dashboards.release-notes-1.3.11.md

```diff
@@ -0,0 +1,11 @@
+# Version 1.3.11 Release Notes
+
+### 🛡 Security
+
+### 📈 Features/Enhancements
+
+### 🐛 Bug Fixes
```

Comment:
This is missing the actual changes of this release: https://github.com/opensearch-project/OpenSearch-Dashboards/pulls?q=is%3Apr+label%3Av1.3.11+is%3Aclosed+base%3A1.3